screens/ProfSignUp.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { View, Text, StyleSheet, TouchableOpacity, ScrollView, Image } from "react-native";
+import { ButtonUi } from "../components/ui/Button";
+import { InputForm } from "../components/ui/InputForm";
+import { useNavigation } from '@react-navigation/native';
+import RadioGroup from 'react-native-radio-buttons-group';
+import { useMemo, useState } from "react";
+
+
+const ProfSignUp = () => {
+
+  return (
+    <View style={styles.screen}>
+      <View style={styles.card}>
+        <View style={styles.centeredContainer}>
+          <Image
+            style={{ marginBottom: 30 }}
+            source={require('../assets/Logo.png')}
+          />
+          <Text style={styles.signUpText}>
+            สมัครสมาชิก
+          </Text>
+        </View>
+        <InputForm title={"คำนำหน้าขื่อ *"} />
+        <InputForm title={"อีเมล *"} />
+        <InputForm title={"รหัสประจำตัวอาจารย์ *"} />
+        <InputForm title={"มหาวิทยาลัยที่จบการศึกษา *"} />
+        <InputForm title={"ประเทศที่จบการศึกษา"} />
+
+        <ButtonUi title={"สมัครสมาชิก"} />
+        <View
+          style={{ flexDirection: "row", alignItems: "center", marginTop: 15, justifyContent: 'center', paddingBottom: 50 }}
+        >
+          <Text style={{ fontSize: 14 }}>สมัครสมาชิกแล้ว? </Text>
```

Comment:
ขนาดเล็กไปใหญ่สุดจะมี sm -> md -> lg -> xl ->2xl...
ตั้งชื่อตามขนาดเป็นกลุ่มของ css
```suggestion
          <Text style={{fontSm }}>สมัครสมาชิกแล้ว? </Text>
```